ui/package.json

```diff
@@ -29,7 +29,7 @@
     "@splunk/react-icons": "^4.4.0",
     "@splunk/react-page": "^7.0.0",
     "@splunk/react-toast-notifications": "^0.11.3",
-    "@splunk/react-ui": "4.28.2",
+    "@splunk/react-ui": "^4.30.0",
```

Comment:
as mentioned on slack lets make that one fixed version
```suggestion
    "@splunk/react-ui": "4.30.0",
```

Reply:
Made the changes.